### ISSUE TITLE
Manually changed learning rate - logger shows something different

### ISSUE THREAD

**Issue by aliemil** [DLR-RM/stable-baselines3#942]
Hello,
Before I set up a proper learning rate schedule I tried to change my learning rate through model.learning_rate = 0.001 and looped over model.learning_rate *= 0.5 after a certain amount of training steps. 
To my surprise the logger shows constantly a learning rate is ~4.3e-9 which just doesnt make sense. 
Is there something  I am missing?

![image](https://user-images.githubusercontent.com/74488721/174910633-690f83a5-b521-434a-980b-def15e3287d6.png)
![image](https://user-images.githubusercontent.com/74488721/174910940-257998a0-c193-4e8f-9d99-ee6d968b184c.png)

**Comment by araffin** [comment]
see https://github.com/DLR-RM/stable-baselines3/issues/735#issuecomment-1047638011 and doc (you should use learning rate schedule, we have an example for that.)

Duplicate of #735 and #856

**Comment by araffin** [comment]
for the value you see,  ~4.3e-9 is basically zero (not exactly because of floating point error), which is not surprising if you divide the lr by two everytime.

**Comment by aliemil** [comment]
The thing is that I divide just two times by 2, not every iteration/epoch/etc. 
And it was just the second iteration.

Schedule works of course much better, I was just wondering

**Comment by araffin** [comment]
as you didn't give a minimal example to reproduce the issue, it's hard to tell what went wrong (the value you fed at init is missing).
